apps/src/studio/constants.js

```diff
@@ -404,3 +404,8 @@ export const SOUND_THROTTLE_TIME = 200;
 // How many milliseconds to throttle between whenTouchObstacle events when
 // blockMovingIntoWalls is enabled.
 export const TOUCH_OBSTACLE_THROTTLE_TIME = 330;
+
+export const BEHAVIOR_CHASE = 'chase';
+export const BEHAVIOR_FLEE = 'flee';
+export const BEHAVIOR_STOP = 'none';
+export const BEHAVIOR_WANDER = 'roam';
```

Comment:
That's the word Brendan used for starwars, and Item.js actually checks for the string `'roam'` when deciding how to move the sprites around.

I now realize that I should have stuck with 'roam' everywhere when I did the spritegroup blocks, changing only the block message to "wander". Now we have a weird mix of both.